rabbitmq/CHANGELOG.md

```diff
@@ -1,14 +1,20 @@
 # CHANGELOG - rabbitmq
 
 
+1.3.2 / Unreleased
+==================
+
+### Changes
+
+* [FEATURE] Add a metric illustrating the available disk space. See [#902][]. (Thanks [@dnavre][])
```

Comment:
There's no actual references to these two at the bottom of the changelog (https://github.com/DataDog/integrations-core/pull/902/files#diff-2b128c36adc27eb29a97dba4938f8e88R49). You can either add them manually, or use [pimpmychangelog](https://github.com/pcreux/pimpmychangelog) to achieve this in a more automated way.